### DISCUSSION TITLE
yubikey commit signing

### DISCUSSION THREAD

**Discussion by cockytrumpet** [NeogitOrg/neogit#891] [Q&A]
Everything is working, this is more of an annoyance. I'm using a yubikey to sign my commits. While I'm typing my pin/touching the key, these messages get spammed to NeogitConsole:

`> git --no-pager -c color.ui=always --no-optional-locks describe --long --tags HEAD
fatal: No names found, cannot describe anything.
`

I tried this in my lazy config to stop the console window from opening:

```lua
  {
    "NeogitOrg/neogit",
    dependencies = {
      "nvim-lua/plenary.nvim", -- required
      "nvim-telescope/telescope.nvim", -- optional
      "sindrets/diffview.nvim", -- optional
      "ibhagwan/fzf-lua", -- optional
    },
    config = true,
    cmd = { "Neogit" },
    setup = function()
      require("neogit").setup {
        auto_show_console = false,
      }
    end,
  },
```

but it's still opening. Any suggestions?

**Comment by ten3roberts** [comment]
I think that may just be another error if you don't have any tags in your commit history.

I'll check it on my machine later and see if I can reproduce.

Is it only in the split console, or do they show as notifications?

**Reply by cockytrumpet** [reply]
It's only in the split console.

Edit: I realized that I was configuring neogit improperly. Now that auto_show_console is actually being set to false, the errors are notifications.

```lua
{
    "NeogitOrg/neogit",
    dependencies = {
      "nvim-lua/plenary.nvim", -- required
      "nvim-telescope/telescope.nvim", -- optional
      "sindrets/diffview.nvim", -- optional
      "ibhagwan/fzf-lua", -- optional
    },
    config = true,
    opts = function()
      local config = require "neogit.config"
      config.auto_show_console = false
      return config
    end,
    cmd = { "Neogit" },
    setup = function()
      require("neogit").setup {}
    end,
  }
```
![Screenshot 2023-10-16 at 12 16 21 AM](https://github.com/NeogitOrg/neogit/assets/3034594/6c2a5a55-f627-4269-af32-6645139dd614)

**Comment by willvio** [comment]
I got the same message everytime a commit is pushed
